psi4/src/read_options.cc

```diff
@@ -1700,8 +1700,8 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         options.add_double("DFT_OMEGA_C", 0.0);
         /*- The DFT Correlation hybrid parameter -*/
         options.add_double("DFT_ALPHA_C", 0.0);
-        /*- Minima rho cutoff for the second derivative -*/
-        options.add_double("DFT_V2_RHO_CUTOFF", 1.e-6);
+        /*- Minima spin-summed density cutoff for the second derivative. Defaults to the density tolerance. -*/
+        options.add_double("DFT_V2_RHO_CUTOFF", -1.0);
```

Comment:
Maybe `0.0` would be a better default? Also, one should note here that Libxc does internal screening.. did the changing the density tolerance change the Libxc threshold as well?

Reply:
I don't understand : what changing the density tolerance? Your use of past tense makes me think you're referring to some change in this PR, but no part of this PR matches that description.

Reply:
> If one changes the density tolerance, does the internal Libxc screening threshold also get changed?

[Yes.](https://github.com/psi4/psi4/blob/33eb8fcd3163f1b77dc3edb63b87cc43d9cf0d5f/psi4/driver/procrouting/dft/superfunctionals.py#L81-L84) That is the one job of the "DFT_DENSITY_TOLERANCE" keyword.

Reply:
And because I forgot to bring this up - the "DFT_DENSITY_TOLERANCE" defaults to -1, which is why this keyword is defaulting to -1.